pyproject.toml

```diff
@@ -56,6 +56,7 @@ markers = [
   "arm_slow: mark a test as slow for arm64 architecture",
   "arraymanager: mark a test to run with ArrayManager enabled",
 ]
+asyncio_mode = "strict"
```

Comment:
where do we use asyncio?

Reply:
We have some tab completion tests that use this

```
(pandas-dev) ...  % grep --include="*.py" "async def" pandas/tests -r
pandas/tests/resample/test_resampler_grouper.py:async def test_tab_complete_ipython6_warning(ip):
pandas/tests/frame/test_api.py:    async def test_tab_complete_warning(self, ip, frame_or_series):
pandas/tests/arrays/categorical/test_warnings.py:    async def test_tab_complete_warning(self, ip):
pandas/tests/indexes/test_base.py:    async def test_tab_complete_warning(self, ip):
```